src/ape/options.py

```diff
@@ -31,3 +36,31 @@ def verbose_option(help=""):
         default=False,
         help=help,
     )
+
+
+class Alias(click.Choice):
+    """Wraps ``click.Choice`` to load account aliases for the active project at runtime."""
+
+    name = "alias"
+
+    def __init__(self, account_type: Type[AccountAPI] = KeyfileAccount):
+        # NOTE: we purposely skip the constructor of `Choice`
+        self.case_sensitive = False
+        self._account_type = account_type
+
+    @property
+    def choices(self) -> List[str]:  # type: ignore
+        # NOTE: This is a hack to lazy-load the aliases so CLI invocation works properly
+        return [a.alias for a in accounts.get_accounts_by_type(self._account_type)]
```

Comment:
Now, the choice aliases and confound to a specific type.
Thus, in the `ape_accounts` plugin, we limit them to `KeyfileAccounts`.

This affords the `delete` command to not be able to delete accounts from other plugins (if they happen to have a `delete()` method implemented).

Reply:
Curious if we might want an `--all` flag to display all the accounts w/ their plugin

Reply:
I could get behind that.